### PR TITLE
Parse GET family

### PR DESCRIPTION
**Pull request by Claes65**

### CODE REVIEW

**Comment by Claes65** [comment]
How can I test mutations()? They seem to be checked elsewhere, but are still flagged here.

File: libs/natparse/src/test/java/org/amshove/natparse/parsing/StatementListParserShould.java [hunk 1]
```diff
@@ -1213,6 +1213,114 @@ void parseReadWorkWithAdjustWithoutAnyKeywordBeforeOperand4()
...
+	@ValueSource(strings =
+	{
+		"#VAR1 #VAR2 #VAR3",
+		"#VAR1 #VAR2 #VAR3(1)",
+		"DATA #VAR1 #VAR2 #VAR3",
+		"DATA #VAR1 #VAR2 #VAR3(1)",
+	})
+	void parseGetTransactionStatements(String statement)
+	{
+		var get = assertParsesSingleStatement("GET TRANSACTION %s".formatted(statement), IGetTransactionNode.class);
```
[the remark this answers is not in the document]
Comment (Claes65) [reply libs/natparse/src/test/java/org/amshove/natparse/parsing/StatementListParserShould.java:1226]: I thought so too :) Let me try again.

[the remark this answers is not in the document]
Comment (Claes65) [reply libs/natparse/src/test/java/org/amshove/natparse/parsing/StatementListParserShould.java:1226]: assertThat(assertNodeType(get.mutations().get(0), IVariableReferenceNode.class).referencingToken().symbolName()).isEqualTo("#VAR1");
                                             ^
  symbol:   method mutations()
  location: variable get of type IGetTransactionNode
1 error

File: libs/natparse/src/main/java/org/amshove/natparse/natural/IGetTransactionNode.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package org.amshove.natparse.natural;
+
+import org.amshove.natparse.ReadOnlyList;
+
+public interface IGetTransactionNode extends IStatementNode
```
Comment (MarkusAmshove) [inline libs/natparse/src/main/java/org/amshove/natparse/natural/IGetTransactionNode.java:5]: this needs to `extends IStatementNode, IMutateVariables`, then you can test it :)

Comment (Claes65) [reply libs/natparse/src/main/java/org/amshove/natparse/natural/IGetTransactionNode.java]: of course, thanks
